v2/bundle/jwtbundle/bundle.go

```diff
@@ -83,72 +83,72 @@ func (b *Bundle) TrustDomain() spiffeid.TrustDomain {
 	return b.trustDomain
 }
 
-// JWTKeys returns the JWT keys in the bundle, keyed by key ID.
-func (b *Bundle) JWTKeys() map[string]crypto.PublicKey {
+// JWTAuthorities returns the JWT authorities in the bundle, keyed by authority ID.
+func (b *Bundle) JWTAuthorities() map[string]crypto.PublicKey {
 	b.mtx.RLock()
 	defer b.mtx.RUnlock()
 
-	return jwtutil.CopyJWTKeys(b.jwtKeys)
+	return jwtutil.CopyJWTAuthorities(b.jwtAuthorities)
 }
 
-// FindJWTKey finds the JWT key with the given key id from the bundle. If the key
+// FindJWTAuthorities finds the JWT authority with the given authority id from the bundle. If the authority
 // is found, it is returned and the boolean is true. Otherwise, the returned
 // value is nil and the boolean is false.
-func (b *Bundle) FindJWTKey(keyID string) (crypto.PublicKey, bool) {
+func (b *Bundle) FindJWTAuthorities(authorityID string) (crypto.PublicKey, bool) {
```

Comment:
Since a single authority is returned, this should be expressed in singular, not plural.